### PR TITLE
Display System: Enabling Text mode debugging

### PR DESCRIPTION
**Pull request by supercomputer7**
We talked about such feature in the IRC channel.
I added an option to start Serenity with text mode in QEMU in the run script.
The boot parameter is very simple for now - "text_debug" without anything further.

### CODE REVIEW

File: Kernel/init.cpp [hunk 1]
```diff
@@ -68,6 +68,14 @@ VFS* vfs;
     auto dev_random = make<RandomDevice>();
     auto dev_ptmx = make<PTYMultiplexer>();
 
+    bool textmode;
+    auto vidmode = KParams::the().get("text_debug");
+    if (vidmode.is_null()) {
```
Comment (Pulgovisk) [inline Kernel/init.cpp]: textmode = !vidmode.is_null();

Comment (supercomputer7) [reply Kernel/init.cpp]: Thanks for the review!
I fixed that.

**Comment by supercomputer7** [comment]
> Good idea, let's just fix some little things :)

Thanks for the review!

**Comment by awesomekling** [comment]
LGTM

**Pull request merged**